export/hgnc_symbol_map.csv

```diff
@@ -1753,8 +1756,8 @@ PODNL1,26275
 POFUT1,14988
 POFUT2,14683
 POGLUT1,22954
-POGLUT2,19350
-POGLUT3,28496
+POGLUT2,None
+POGLUT3,None

```

Comment:
I wonder what happened here, I get proper IDs for these genes, I'll run the export myself to update this.